pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "ya-aioclient"
-version = "0.5.0"
+version = "0.6.1"
```

Comment:
Is skipping both minor and patch on purpose here?

Reply:
Release 0.6.0 was yanked